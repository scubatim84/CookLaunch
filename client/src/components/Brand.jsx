import React from 'react';
import {Link, Typography} from '@material-ui/core';

function Brand() {
  return (
    <Link href='/'>
      <Typography variant='h4' color='textPrimary'>
        Cook Launch
      </Typography>
    </Link>
  );
}

export default Brand;
