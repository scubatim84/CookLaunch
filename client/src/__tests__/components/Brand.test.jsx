import React from 'react';
import {shallow} from 'enzyme';
import Brand from '../../components/Brand';

describe('Brand', () => {
  it('Renders correctly', () => {
    shallow(<Brand />);
  });
});
