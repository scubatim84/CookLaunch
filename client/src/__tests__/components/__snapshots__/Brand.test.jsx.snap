import React from 'react';
import {shallow} from 'enzyme';
import Brand from '../../components/Brand';

describe('Brand', () => {
  it('Renders component without crashing', () => {
    shallow(<Brand />);
  });
});
